src/index.js

```diff
@@ -0,0 +1,46 @@
+import './index.css';
+import list from './list';
+// check icon
+//
+// trash icon
+// <i class="fa-regular fa-trash-can"></i>
```

Comment:
- [optional] please always have in mind to remove commented code. This will keep your code clean and professional. Nice work